WireDomain/Tests/WireDomainTests/Repositories/UserRepositoryTests.swift

```diff
@@ -149,6 +204,18 @@ class UserRepositoryTests: XCTestCase {
             supportedProtocols: [.mls],
             legalholdStatus: .disabled
         )
+
+        nonisolated(unsafe) static let event = UserUpdateEvent(
+            id: userID,
+            userID: UserID(uuid: userID, domain: domain),
+            accentColorID: nil,
+            name: "username",
+            handle: "test",
+            email: "test@wire.com",
+            isSSOIDDeleted: nil,
+            assets: nil,
+            supportedProtocols: [.proteus, .mls]
+        )
```

Comment:
I think it may be good to also have some more varying data, with some more `nil`, so see that only new changes are persisted and old values are preserved in case of `nil` values.

Reply:
done in https://github.com/wireapp/wire-ios/pull/1958/commits/411d1ab958b6f0aa0735b39903db173c84783820